app/src/features/apiClient/screens/apiClient/components/sidebar/components/collectionsList/requestRow/RequestRow.scss

```
@@ -31,3 +40,19 @@ svg.graphql-request-icon {
   border-bottom: 2px solid var(--requestly-color-primary-text, #1890ff);
   margin-bottom: -2px;
 }
+
+.collapsible-request-row-header {
+  display: flex;
+  align-items: center;
+  gap: var(--requestly-space-2, 4px);
+  // padding: var(--space-2, 4px);
```

Stylelint (17.3.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)